tests/config-test.js

```diff
@@ -75,6 +78,7 @@ describe('src/config', function () {
           creditLimit: 1,
           resetSeconds: 60,
           actorField: '',
+          label: null,
```

Comment:
most configs won't explicitly set `label` to `null`. 
maybe specify one as `undefined` and another with no key for `label` for good measure?